fapi-client/src/main/scala/com/gu/facia/api/utils/Backfill.scala

```diff
@@ -40,10 +39,11 @@ object BackfillResolver {
         } yield {
           backfillContent.map(CuratedContent.fromTrailAndContent(_, TrailMetaData.empty, None, collectionConfig))
         }
-      case CollectionBackfill(parentCollectionId) =>
+      case CollectionBackfill(parentCollectionId) => {
         for {
           collection <- FAPI.getCollection(parentCollectionId)
           curatedCollection <- FAPI.liveCollectionContentWithSnaps(collection, adjustSearchQuery, adjustItemQuery)
         } yield curatedCollection
+      } recover { err => List() }
```

Comment:
@janua Ok, I'll do this. Can you give me an example of where logging is done in facia tool?